templates/base/Dockerfile

```diff
@@ -3,10 +3,35 @@ FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes build-essential ca-certificates curl jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip && \
 	apt-get upgrade --yes && \
 	apt-get clean
 
+	RUN VERSION=8.7.1 && \
+	curl https://curl.se/download/curl-${VERSION}.tar.gz --output curl-${VERSION}.tar.gz && \
+	tar -xzvf curl-${VERSION}.tar.gz && \
+	rm -f curl-${VERSION}.tar.gz && \
+	cd curl-${VERSION} && \
+	apt-get remove curl -y && \
+	rm -rf /usr/bin/curl* && \
+	./configure --prefix=/usr/local --with-ssl --with-nghttp2 --enable-static && \
+	make -j4 && \
+	make install && \
+	ldconfig && \
+	cd .. && \
+	rm -rf curl-${VERSION}
+
+RUN VERSION=5.2 && \
+	curl https://ftp.gnu.org/gnu/bash/bash-${VERSION}.tar.gz --output bash-${VERSION}.tar.gz && \
+	tar -xzvf bash-${VERSION}.tar.gz && \
+	rm -f bash-${VERSION}.tar.gz && \
+	cd bash-${VERSION} && \
+	./configure --prefix=/usr && \
+	make -j4 && \
+	make install && \
+	cd .. && \
+	rm -rf bash-${VERSION}
+
```

Comment:
Hey @Wykenz, can you please create a new ticket to track this fix and use as the commit message prefix?

Ideally, we shouldn't use LPPs in the commit messages since they're for internal usage only. :slightly_smiling_face: